<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FInBoxDialog</class>
 <widget class="QDialog" name="FInBoxDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FInBoxDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>f_in_box</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox_flashocx_path">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>F:\zhly_pt\trunk\src\client\cs\zly_install_shell\ZhlyResBase\flash.ocx</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>待打开的flash文件的路径:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>flash.ocx路径:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QComboBox" name="comboBox_flashfile_path">
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>F:\document\1201a\start.swf</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButton_open_flash_use_f_in_box">
        <property name="text">
         <string>使用f_in_box,加载flash.ocx到内存,打开flash文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
